package.json

```diff
@@ -15,6 +15,15 @@
     "chart.js": "2.9.4",
     "classnames": "2.3.1",
     "core-js": "3.21.1",
+    "@types/color-hash": "1.0.2",
+    "@types/country-list": "2.1.0",
+    "@types/react-overlays": "1.1.3",
+    "bootstrap": "3.4.1",
```

Comment:
I think some of these packages were purposely removed, can we rebase from master and make sure we're only adding the packages needed for this change?

Reply:
Oh duh. I actually thought I fixed these. I'll rebase and fix again.